stripe/createCheckoutSession.js

```diff
@@ -52,9 +82,11 @@ export const main = async event => {
   }
 
   console.log('Initialize Stripe');
+  /** @type {Stripe} */
   const stripe = Stripe(stripeSecrets.API_SECRET_KEY);
 
   console.log('Create a payment checkout session with Stripe');
+  /** @type {Stripe.Checkout.Session} */
```

Comment:
If I recall correctly, I have added `@types/stripe` as a dependency. And I think VSCode reads from the package. Do we really need to add these comments, when they already exist?

Reply:
the variable `session` declared without assignment. it means later on anything could be assigned to this variable. static code analyzer could not be sure what is the type of the variable. that is why we need to help it and declare a type to leveraging the intellisense and other features provided by typing